### PR TITLE
🏗 Optionally exclude CSS from generated extension

### PR DESCRIPTION
**Pull request by alanorozco**
Adds `amp make-extension --nocss` flag to exclude generating CSS and referencing it.

### CODE REVIEW

**Comment by alanorozco** [comment]
New directory structure:
```
build-system/tasks/make-extension/template/
├── bento
│   └── extensions
│       └── amp-__component_name_hyphenated__
│           ├── OWNERS
│           └── __component_version__
│               ├── amp-__component_name_hyphenated__.js
│               ├── base-element.js
│               ├── component.js
│               ├── component.type.js
│               ├── storybook
│               │   ├── Basic.amp.js
│               │   └── Basic.js
│               └── test
│                   └── test-amp-__component_name_hyphenated__.js
├── classic
│   ├── examples
│   │   └── amp-__component_name_hyphenated__.html
│   └── extensions
│       └── amp-__component_name_hyphenated__
│           ├── OWNERS
│           └── __component_version__
│               ├── amp-__component_name_hyphenated__.js
│               └── test
│                   ├── __validator__-amp-__component_name_hyphenated__.html
│                   └── test-amp-__component_name_hyphenated__.js
├── css
│   └── extensions
│       └── amp-__component_name_hyphenated__
│           └── __component_version__
│               └── amp-__component_name_hyphenated__.css
├── jss
│   └── extensions
│       └── amp-__component_name_hyphenated__
│           └── __component_version__
│               └── component.jss.js
└── shared
    └── extensions
        └── amp-__component_name_hyphenated__
            ├── __validator__-amp-__component_name_hyphenated__.protoascii
            └── amp-__component_name_hyphenated__.md
 ```

File: build-system/tasks/make-extension/index.js [hunk 1]
```diff
@@ -204,19 +205,44 @@ async function makeExtensionFromTemplates(
   const version = (
     options.version || (options.bento ? '1.0' : '0.1')
   ).toString();
+  const {'no_css': noCss} = options;
```
Comment (caroqliu) [inline build-system/tasks/make-extension/index.js:208]: In an effort to avoid negative boolean names, WDYT about `has_css` default to `true`? go/tott-530

**Review by caroqliu — APPROVED** [review]
Thanks for doing this!

File: build-system/tasks/make-extension/index.js (hunk 1 above)
Comment (alanorozco) [reply build-system/tasks/make-extension/index.js:208]: Unfortunately [minimist](https://www.npmjs.com/package/minimist) lacks a way to set arguments as `false`, it merely treats their presence as true. Hence all these flags prefixed `no`:

```
build-system/tasks/build.js:
  99:   noextensions: 'Builds with no extensions.',

build-system/tasks/default-task.js:
  76:   noconfig: 'Compiles production binaries without applying AMP_CONFIG',

build-system/tasks/dist.js:
  390:   noconfig: 'Compiles production binaries without applying AMP_CONFIG',
  395:   noextensions: 'Builds with no extensions.',

build-system/tasks/coverage-map/index.js:
  176:   nobuild: 'Skips dist build.',

build-system/tasks/runtime-test/helpers.js:
  86:     nobuild: 'Skipping build.',
```

Comment (caroqliu) [reply build-system/tasks/make-extension/index.js]: Ahh, thanks for the context!

**Pull request merged**